Scala/syntax_test_scala.scala

```diff
@@ -1189,3 +1189,38 @@ def <(a: Int) = 42
      &amp;
 //   ^^^^^ constant.character.entity.xml - meta.tag.xml
    </foo>
+
+// annotation examples from: http://www.scala-lang.org/files/archive/spec/2.11/11-annotations.html
+@deprecated("Use D", "1.0") class C { ... }
+// ^^ storage.modifier.annotation
+//            ^^ string
+
+@transient @volatile var m: Int
```

Comment:
Add assertions for the second annotation here to ensure we a properly allowing two back-to-back.